test/test-find-python.js

```diff
@@ -17,8 +16,8 @@ test('find python', function (t) {
     t.strictEqual(err, null)
     var proc = execFile(found, ['-V'], function (err, stdout, stderr) {
       t.strictEqual(err, null)
-      t.strictEqual(stdout, '')
-      t.ok(/Python 2/.test(stderr))
+      t.ok(stdout === '' || stderr === '')
+      t.ok(/Python 2/.test(stderr) || /Python 3/.test(stdout))
```

Comment:
Non-blocking but I think these two tests are going to be hard to debug if they fail (in the previous code if `stdout` wasn't equal to `''` the contents would be included in the test failure).

Reply:
Would it be possible to refactor this so that if we're on Python 3, we run checks for that, which are strict and so output contents if they fail, otherwise we run stricts for Python 2?

Reply:
Updated, PTAL.